advanced_function_usage/HW6_task2.js

```diff
@@ -0,0 +1,10 @@
+function printNum(num) {
+    console.log(num);
+    if (num > 0) {
+        printNum(num - 1);
+    } else {
```

Comment:
Краще скоротити кількість if... else, саме тут можна поміняти трошки функцію, аби вона мала вигляд:
```suggestion
    console.log(num);
    if (num <= 0) {
        return;
    }
    printNum(num - 1);
```

Reply:
Fixed